### PR TITLE
fix #81170 simplyfaucet.xyz

### LINKED ISSUE #81170: simplyfaucet.xyz
### Issue URL (Anti Adblock Script)
 [https://simplyfaucet.xyz/?r=LMKpMZJoDD3GFEdGg5pQBsQRndymiZoSGk](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fsimplyfaucet.xyz%2F%3Fr%3DLMKpMZJoDD3GFEdGg5pQBsQRndymiZoSGk)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.34
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (2880), <br>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection, <br/>EasyPrivacy<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish, <br/>AdGuard Russian, <br/>AdGuard Spanish/Portuguese, <br/>EasyList Italy

### PR DESCRIPTION
**Pull request by stanislav-atr**
#81170

### CODE REVIEW

File: EnglishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -617,6 +617,8 @@ jacquieetmicheltv.net#%#//scriptlet('set-constant', 'is_adblocked', 'false')
 ||hentaisvision.biz/js/adblock.js
 ! https://github.com/AdguardTeam/AdguardFilters/issues/80870
 movieverse.co#@#.ads_single_center
+! https://github.com/AdguardTeam/AdguardFilters/issues/81170
+@@||simplyfaucet.xyz/libs/advertisement.js$domain=simplyfaucet.xyz
```
Comment (Sergey-Lyapin) [inline EnglishFilter/sections/antiadblock.txt:621]: please add to the existing rule:
`@@/libs/advertisement.js`

File: EnglishFilter/sections/specific.txt [hunk 2]
```diff
@@ -262,7 +262,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
...
...ovieshd.cloud|123movieshd.*|voxc.org|link.bitmos.co.in|movie8k.*|sports24.*|shortenbuddy.com|mac-torrent-download.net|openloadmov.net|downloadpirate.com|payskip.org|moviesflix.one|themoviesflix.co|hentaitube.online|bestshort.xyz|link1s.*|earnwithshortlink.com|foxseotools.com|manhuascan.com|exe.app|10convert.com|crownimg.com|fc.lc|manganelo.com|mypornstarbook.net|7hitmovies.surf|desitvshows.xyz|kissaway.net|strims.world|srek.net|cpmlink.net|shrinkzzy.link|cookuve.com|bigtitslust.com|sanos.xyz|holymanga.net|links.mflixblog.xyz|skidrowreloaded.com|lovehug.net|pewgame.com|himovies.to|warefree01.com|kissanimes.cc|toonanime.tv|sxyprn.com|anupama.*|shortlinkpay.com|kiryuu.co|blurayufr.com|btcads24.com|forex-trnd.com|wolfstream.tv|tuktukcinema.net|mynewshub.in|mynewsmedia.co|userupload.net|crypto-faucet.xyz|doctor-groups.com|downloadhub.kim|myadultanimes.com|otakudesu.moe|freeadultcomix.com|shrlink.top|mfk-shorter.com|lelscan-vf.co|ruvideos.net|srsone.top|animexin.xyz|otakufr.co|4anime.to|french-stream.lol|jujmanga.com|watchseries.*|ahwaktv.tv|allviids.com|amlijatt.in|fulltube.online|123movies.*|bevru.club|lewdweb.net|shortenmm.cf|fsapi.xyz|flixtv.club|worldcupfootball.me|promo-visits.site
```
Comment (Sergey-Lyapin) [inline EnglishFilter/sections/specific.txt:265]: please add adservers to adservers.txt

File: EnglishFilter/sections/specific.txt [hunk 3]
```diff
@@ -262,7 +262,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
...
...movieshd.*|voxc.org|link.bitmos.co.in|movie8k.*|sports24.*|shortenbuddy.com|mac-torrent-download.net|openloadmov.net|downloadpirate.com|payskip.org|moviesflix.one|themoviesflix.co|hentaitube.online|bestshort.xyz|link1s.*|earnwithshortlink.com|foxseotools.com|manhuascan.com|exe.app|10convert.com|crownimg.com|fc.lc|manganelo.com|mypornstarbook.net|7hitmovies.surf|desitvshows.xyz|kissaway.net|strims.world|srek.net|cpmlink.net|shrinkzzy.link|cookuve.com|bigtitslust.com|sanos.xyz|holymanga.net|links.mflixblog.xyz|skidrowreloaded.com|lovehug.net|pewgame.com|himovies.to|warefree01.com|kissanimes.cc|toonanime.tv|sxyprn.com|anupama.*|shortlinkpay.com|kiryuu.co|blurayufr.com|btcads24.com|forex-trnd.com|wolfstream.tv|tuktukcinema.net|mynewshub.in|mynewsmedia.co|userupload.net|crypto-faucet.xyz|doctor-groups.com|downloadhub.kim|myadultanimes.com|otakudesu.moe|freeadultcomix.com|shrlink.top|mfk-shorter.com|lelscan-vf.co|ruvideos.net|srsone.top|animexin.xyz|otakufr.co|4anime.to|french-stream.lol|jujmanga.com|watchseries.*|ahwaktv.tv|allviids.com|amlijatt.in|fulltube.online|123movies.*|bevru.club|lewdweb.net|shortenmm.cf|fsapi.xyz|flixtv.club|worldcupfootball.me|promo-visits.site|simplyfaucet.xyz
```
Comment (Sergey-Lyapin) [inline EnglishFilter/sections/specific.txt:265]: and this rule breaks captcha

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/specific.txt (hunk 3 above)
Comment (Sergey-Lyapin) [reply EnglishFilter/sections/specific.txt:265]: let's add raincaptcha.com as an exclusion in this regexp (and in the same regexps in other files)

**Review by zloyden — APPROVED** [review]

**Pull request closed without merging**